lib/formatters/githubFormatter.mjs

```diff
@@ -23,6 +23,8 @@ export default function githubFormatter(results, returnValue) {
 		});
 	});
 
+	if (lines.length) lines.push(`::notice title=${title}::The github formatter is deprecated`);
```

Comment:
```suggestion
	lines.push(`::notice title=${title}::The github formatter is deprecated`);
```

I think we always want to emit this notice.
Not only when there are other warnings.

Reply:
Check the `outputs no warnings` tests.
i.e. all the formatters `output === ""` when `warnings.length === 0`

Reply:
I found another formatter—`verbose`—that returns a truthy value when `warnings` array is empty so I will remove the `if`.
I still think it's a bit risky and will have unintended/unknown consequences.

Reply:
If it draws peoples attention to this formatter and makes them consider the upcoming deprecation then maybe those are still good consequences 😛 

If there really is something major that we could not have foreseen then we can revert the change and regroup from there.

Reply:
> If there really is something major that we could not have foreseen then we can revert the change and regroup from there.

That's what I was trying to avoid.
i.e. no regression test has been foreseen